spec/models/user_spec.rb

```diff
@@ -22,5 +22,9 @@
 
       it { is_expected.to_not be_valid }
     end
+
+    describe 'database columns' do
```

Comment:
Specs are usually a nice thing, but in this specific case I think we are not testing our functionality, but how rails/migration work. And these are tested by default, with the specs written for the framework we are using ;)